### ISSUE TITLE
Quirks/bugs in Manage Spots and RM

### ISSUE THREAD

**Issue by NicoleRayner** [CIRDLES/Squid#334]
When creating a brand new squid project I cannot select a calibration RM unless I first go to Data - Manage Sample naming first.  The dropdown on Data - Manage Spots and Ref Mats is greyed out until I "activate" it by visiting the Sample Naming page. This is an awkward/non-intuitive workflow but I'm not sure what the best way around this would be.  Have Manage Sample Naming **_within_** Manage Spots and RefMats? 

On a related note, there should probably be a "refresh" button under the RM and CRM model windows in Data - Manage Spots and RefMats.  Just as there is for the Common Pb and Physical models in the Manage Task window.  I made a new model today and it took quite a bit of refreshing (in Task Manager) and opening/saving (directly from Parameters models) to get it visible/accessible to me in the Manage Spots and RM drop down.

**Comment by NicoleRayner** [comment]
I'm just highlighting this issue again (particularly the 2nd part regarding a "refresh" button under the RM and CRM model windows in Data - Manage Spots and RefMats).  It came up for me today using the new version and I had to make some new parameter models.  

Interestingly the new CRM values (conc ppm) made it into the Reference Materials Values without any problem but I had to deselect/reselect my new RM parameter model in order to get the RefRad_Ages values to update.  I haven't attempted to reproduce this glitch however.
